docs/pages/guides/migrating-to-react-native-firebase.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Migrating from Expo Firebase to React Native Firebase
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+
+The `expo-firebase-analytics` and `expo-firebase-recaptcha` packages allowed the use of some Firebase features on Classic Builds, Expo Go, and Expo for Web, but developers often encountered version conflicts when trying to use them alongside the rest of the Firebase suite. With EAS Build and development builds, it is much easier to use [React Native Firebase](https://rnfirebase.io/) directly in your entire development workflow, so these packages have been deprecated.
+
+In this guide, we will migrate from the Expo Firebase packages to React Native Firebase by:
+- Replacing the Expo Firebase packages with their React Native Firebase equivalents
+- Replacing the use of Expo Go for testing with a [development build](/development/introduction/) customized to your project's needs
+- Testing the updated code on your development build
+
+> **info** These instructions assume you have already been successfully using the Expo Firebase packages. If you're adding Firebase to your app for the first time, see the [Using Firebase](/guides/using-firebase) guide.
+
+## Install React Native Firebase
+Run the following commands to remove `expo-firebase-analytics` and install the React Native Firebase packages:
+
+<Terminal cmd={[
+'# Remove Expo Firebase packages',
+'$ yarn remove expo-firebase-analytics',
+'',
+'# Install RN Firebase, etc.',
+'$ yarn add @react-native-firebase/app @react-native-firebase/analytics'
```

Comment:
use npm instead - it's the default and anyone using yarn will know how to translate to npm